src/js/overlay.js

```diff
@@ -201,8 +201,6 @@ Overlay.prototype.show = function() {
 		document.body.appendChild(shadow);
 	}
 
-	this.content.focus();
```

Comment:
We still need a focus somewhere for accessibility, unless I'm missing something?